nautobot/docs/models/extras/relationships.md

```diff
@@ -0,0 +1,71 @@
+# Relationships
```

Comment:
This file name needs to be `relationship.md` rather than `relationships.md` for the online help auto-linking to work properly.

Reply:
Sorry I am not following this one. Some have the s, some don't, but I have used the plural everywhere? I believe the model filename is also relationships.py
e.g. (which I've roughly followed as a guide for this doc)
`- Custom Fields: 'additional-features/custom-fields.md'`

I should change the file name as well as the links to simply relationship.md?

Reply:
The filename specifically needs to be `relationship.md`. The online help within the Nautobot application looks specifically for a file at `nautobot/docs/models/<app_label>/<modelname>.md` - here, `nautobot/docs/models/extras/relationship.md`.

Current observed behavior when clicking the (?) icon on the "Add a new relationship" form:
![image](https://user-images.githubusercontent.com/5603551/109563191-71140380-7aad-11eb-9dd0-b21d369b4b19.png)


"Relationships" is fine for the text here and elsewhere. It's just the filename that's persnickety.

Reply:
Thank you for the explanation! I do see that the model is Relationship. This makes sense.